### PR TITLE
Support GPU on eksctl created AWS instances

### PR DESCRIPTION
**Pull request by yuvipanda**
- Document howto set up GPUs on AWS
- Temporarily add a GPU profile to the uwhackweeks
  hub, until we setup an account for the snowex hackweek

Ref https://github.com/2i2c-org/infrastructure/issues/1309

### CODE REVIEW

**Comment by yuvipanda** [comment]
@scottyhq is it alright if I deploy a GPU profile to uwhackweeks hub to test it out? Or do you think that will be problematic and people can misuse it?

File: docs/howto/features/gpu.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+   This should output something like:
+
+   ```
+   [PhysicalDevice(name='/physical_device:GPU:0', device_type='GPU')]
+   ```
+4. Remember to explicitly shut down your server after testing,
+   as GPU instances can get expensive!
+
+If either of those tests fail, something is wrong and off you go debugging :)
```
Comment (sgibson91) [inline docs/howto/features/gpu.md:140]: In the future, we should develop a GPU debugging guide

**Review by sgibson91 — APPROVED** [review]

File: docs/howto/features/gpu.md (hunk 1 above)
Comment (yuvipanda) [reply docs/howto/features/gpu.md:140]: I made a note in https://github.com/2i2c-org/infrastructure/issues/1167, @sgibson91

**Comment by yuvipanda** [comment]
I'm going to wait on @scottyhq to see if using the uwhackweeks hub in this way is ok before merging this.

**Comment by scottyhq** [comment]
> @scottyhq is it alright if I deploy a GPU profile to uwhackweeks hub to test it out?  

Definitely OK! Thanks!

**Comment by yuvipanda** [comment]
Thanks @scottyhq! You're welcome to test out any content in the uwhackweeks hub if you wish.

**Comment by scottyhq** [comment]
So far so good! 🎉

Screen Shot 2022-05-18 at 10 19 23 AM

**Pull request merged**